Node.js class 1

Michael-Antczak opened the pull request:
**Note:** If you are adding a new lesson or have edited the name of a lesson file. Please make sure you update `./SUMMARY.md` to reflect the changes in the gitbook contents.

Michael-Antczak committed:
Move server section up

Michael-Antczak committed:
Add "what is node" section

Michael-Antczak committed:
Add what to use Node for

Michael-Antczak committed:
Add link to package.json for Passport

Michael-Antczak committed:
Add simple server section

Michael-Antczak committed:
Add http section

Michael-Antczak committed:
Add http diagram

Michael-Antczak committed:
Update http section with image

Michael-Antczak committed:
Add tutsplus http tutorial

kabaros commented on node-db/lesson1.md:
````diff
@@ -9,8 +9,132 @@
+server.listen(5000); 	
+
+console.log('Node.js web server at port 5000 is running..')
+```
````
it looks great .. could you just use const and let for the http example to be consistent with the rest .. also just have the two routes example (no need for single route imo).

Then maybe have an exercise for them to add a /mentor route

Michael-Antczak committed:
Add requested changes

Changes implemented: 
- var to const
- remove one route example
- add routing exercise

kabaros committed:
Update lesson1.md
